File: Task2/PMRam.py
def gcd(a,b):
    if b==0:
        return a
    else:
        return gcd(b,a%b)
def lcm(a,b):
    return (a*b)//(gcd(a,b))
t = int(input())
result_set = []
for T in range(t):
    n = int(input())
    arr = [int(ele) for ele in input().split()]
    res_found = False
    res = 0
    for i in range(len(arr)):
        for j in range(i+1,len(arr)):
            temp = lcm(arr[i],arr[j])
            if not res_found or temp<res:
                res = temp
                res_found = True
    print(res)

File: Task2/PMRam.java
import java.io.*;
import java.util.*;

public class Solution {
    public static long gcd(long a,long b){
        if(a==0)
            return b;
        else
            return gcd(b%a,a);
    }
    public static long lcm(long a,long b){
        return (a*b)/gcd(a,b);
    }
    public static void main(String[] args) {
        /* Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution. */
        Scanner in = new Scanner(System.in);
        int t = in.nextInt();
        while(t-- > 0){
            int n = in.nextInt();
            int arr[] = new int[n];
            for(int i=0;i<n;i++)
                arr[i] = in.nextInt();
            boolean found = false;
            long res = 0;
            for(int i=0;i<n;i++){
                for(int j=i+1;j<n;j++){
                    long temp = lcm(arr[i],arr[j]);
                    if(!found || temp<res){
                        res = temp;
                        found = true;
                    }
                }
            }
            System.out.println(res);
        }
    }
}